molgenis-app-gavin2/src/main/resources/molgenis_ui.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<molgenis label="OMX" name="omx">
```

Comment:
Is this the desired default gavin app menu?

Reply:
Default menu was ok with Joeri